### PR TITLE
Kafka bindings component add version

### LINKED ISSUE #2139: Support configuring broker version in Kafka bindings component
Add documentation for add  Kafka bindings component configuration metadata properties (version)

### PR DESCRIPTION
**Pull request by light-pan**
Signed-off-by: light-pan <373465009@qq.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

The code parse kafka version in **github.com/Shopify/sarama v1.23.1**  utils.go

```go
func ParseKafkaVersion(s string) (KafkaVersion, error) {
	if len(s) < 5 {
		return MinVersion, fmt.Errorf("invalid version `%s`", s)
	}
	var major, minor, veryMinor, patch uint
	var err error
	if s[0] == '0' {
		err = scanKafkaVersion(s, `^0\.\d+\.\d+\.\d+$`, "0.%d.%d.%d", [3]*uint{&minor, &veryMinor, &patch})
	} else {
		err = scanKafkaVersion(s, `^\d+\.\d+\.\d+$`, "%d.%d.%d", [3]*uint{&major, &minor, &veryMinor})
	}
	if err != nil {
		return MinVersion, err
	}
	return newKafkaVersion(major, minor, veryMinor, patch), nil
}
```

```go
SupportedVersions = []KafkaVersion{
		V0_8_2_0,
		V0_8_2_1,
		V0_8_2_2,
		V0_9_0_0,
		V0_9_0_1,
		V0_10_0_0,
		V0_10_0_1,
		V0_10_1_0,
		V0_10_1_1,
		V0_10_2_0,
		V0_10_2_1,
		V0_11_0_0,
		V0_11_0_1,
		V0_11_0_2,
		V1_0_0_0,
		V1_1_0_0,
		V1_1_1_0,
		V2_0_0_0,
		V2_0_1_0,
		V2_1_0_0,
		V2_2_0_0,
		V2_3_0_0,
	}
```
obviously，the value in yaml is `1.0.0` `1.1.0` and so on

## Issue reference

[2139](https://github.com/dapr/docs/issues/2139)

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md [hunk 1]
```diff
@@ -54,6 +56,7 @@ spec:
 | saslPassword | N | Input/Output | The SASL password used for authentication. Can be `secretKeyRef` to use a [secret reference]({{< ref component-secrets.md >}}). Only required if `authRequired` is set to `"true"`. | `""`, `"KeFg23!"` |
 | initialOffset   | N | The initial offset to use if no offset was previously committed. Should be "newest" or "oldest". Defaults to "newest". | `"oldest"` |
 | maxMessageBytes | N | Input/Output | The maximum size in bytes allowed for a single Kafka message. Defaults to 1024. | `2048` |
+| version | N | Kafka cluster version. Defaults to 1.0.0.0 | `1.0.0`
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md]: Looks like you have an extra 0 here. Should this be "Defaults to 1.0.0"

Comment (light-pan) [reply daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md]: The kafka version is 1.0.0.0. In the Description show why the value in yaml is 1.0.0.

Comment (msfussell) [reply daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md]: I do not understand. Looks like there are two default values here. 1.0.0.0 and 1.0.0. Which is correct?

Comment (light-pan) [reply daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md]: 1.0.0

Comment (msfussell) [reply daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md]: So can you change the 1.0.0.0 to 1.0.0 in the "Binding support" column?

Comment (light-pan) [reply daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md]: OK

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**